front/src/modules/simulationResult/components/SpaceTimeChartWrapper/SpaceTimeChartWrapper.tsx

```diff
@@ -166,16 +167,16 @@ const SpaceTimeChartWrapper = ({
 
     if (
       timetableItemUsedForProjectionWithDetails &&
-      'exceptions' in timetableItemUsedForProjectionWithDetails &&
+      isPacedTrainWithDetails(timetableItemUsedForProjectionWithDetails) &&
       isOccurrenceId(selectedProjectionId)
     ) {
-      const exeptionUsedForProjection = timetableItemUsedForProjectionWithDetails.exceptions.find(
-        (exception) =>
+      const exeptionUsedForProjection =
```

Comment:
Nit: typo (missing 'c')

This was already present before this PR, but since you're touching this file, it might be a good opportunity to fix it. ;)
```suggestion
      const exceptionUsedForProjection =
```